File: ASSIGNMENT 3/sum of digits in anumber.cpp
#include<iostream>
using namespace std;
class Sum_digits
{
int num,sum=0,rem;
public: void check_digits()
{
cout<<"Enter a number: ";
cin>>num;
while(num>0)
{
rem=num%10;
sum=sum+rem;
num=num/10;
}
cout<<"Sum is "<<sum<<endl;

}
};

int main()
{
    Sum_digits s1;
    s1.check_digits();
}

File: ASSIGNMENT 4(java)/sumofdigitsinanumber.java
import java.util.Scanner;
class Sum
{
   public static void main(String args[])
{
      int num,rem;
      int sum=0;
      System.out.println("Enter the number:");
      Scanner sc = new Scanner(System.in);
      num = sc.nextInt();
 
      while(num>0)
      {
        rem=num%10;
        sum=sum+rem;
        num=num/10;

      }
      System.out.println("Sum is: "+sum);
   }
  
   }
